recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
> What version of gcc did you get from yum? The ABI is very tightly tied to the particular version of gcc/gfortran.

I see. I think I have a lot more to learn about versioning here. 😄

This was CentOS 6. So, it is gcc 4.4.7.

> I'll look into it on mac. I'm sure there's a way.

Ok, sounds good.

> I think this is where we can use "filters" or custom functions that we can define on a per-recipe basis to feed into jinja2.

That's an interesting idea. Just want to make sure something appropriate gets installed. `post-link` scripts don't cut it as they would just fail to install if there was a mismatch. We need to have some proper fallback. Not sure how.

If having the same version with different ABI can happen, we need to be able to disambiguate the library itself. Would it be possible to encode this in the build string of `libgfortran` package or is that a bad idea?